src/mlpack/methods/decision_tree/best_binary_numeric_split_impl.hpp

```diff
@@ -103,8 +103,24 @@ double BestBinaryNumericSplit<FitnessFunction>::SplitIfBetter(
       ++classCounts(sortedLabels[i], 1);
   }
 
+  // Flags for validating whether the current sample index k is
+  // eligible as a cut point. {repeat} if the value at index k is
+  // part of a repeating sequence. {class_changed} if the class at 
+  // index k changed from k-1. {eligible_cut} if the value did not 
+  // repeat and the class changed, meaning index k is a boundary 
+  // point. See Fayyad and Irani (1992) for more details on eligible 
+  // cut points.
```

Comment:
Could you change the style of comment to be `/** */`

Reply:
Definitely: I just made a new commit. Does mlpack have a preferred workflow for pull requests?